ledger/ledger-api-test-tool-on-canton/BUILD.bazel

```diff
@@ -64,5 +64,5 @@ conformance_test(
         5021,
     ],
     server = ":canton-test-runner-with-dependencies",
-    tags = ["manual"],
+    tags = [],
```

Comment:
If we really want to have this run on CI we can simply remove this line. @SamirTalwar?

Reply:
Ooops, didn't notice he already reviewed. :sweat_smile: Resolving.

Reply:
> If we really want to have this run on CI we can simply remove this line. @SamirTalwar?

👍 - I will try it out and make sure that bazel defaults this properly and does not get upset.

Reply:
Oh, good point, we can just delete this line.